array/js/array.js

```diff
@@ -0,0 +1,92 @@
+function isArray(object) {
+  return Array.isArray(object);
```

Comment:
it's cool you know Array method and can use it, but I want to see your implementation